rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/JacksonCustomizations.java

```diff
@@ -2167,7 +2167,9 @@ public static abstract class ClearingRequestMixin extends ClearingRequest {
                 "setText",
                 "setCommentedBy",
                 "setCommentedOn",
-                "setAutoGenerated"
+                "setAutoGenerated",
+                "setDateTime",
+                "setUsername"
```

Comment:
Would `dateTimeIsSet` and `usernameIsSet` need to be ignored as well?

Reply:
these fields are not present therefore no need to ignore.